src/stateClass.ts

```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Neil Enns. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+enum StateClass {
+  MEASUREMENT = "measurement"
```

Comment:
```suggestion
  MEASUREMENT = "measurement",
  TOTAL = "total",
  TOTAL_INCREASING = "total_increasing"
```

Reply:
Might as well list all three in the enum just in case in the future one of them is needed.